src/services/data.ts

```diff
@@ -0,0 +1,34 @@
+import { Status } from '../types/Status';
+import { Todo } from '../types/Todo';
+
+export function data(
+  todos: Todo[],
+  filterValues: Status,
+  query: string,
+): Todo[] {
+  let preparedTodos = [...todos];
```

Comment:
you can move it outside component file as function.
If you can write constant instead let use it prefer.
let worsens the readability and perception of the code during scaling

Reply:
fixed